e2e/pnpm_workspace_rerooted/MODULE.bazel

```diff
@@ -0,0 +1,60 @@
+module(
+    name = "e2e_pnpm_workspace_rerooted",
+    version = "0.0.0",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_nodejs", version = "5.5.3")
+bazel_dep(name = "aspect_rules_js", version = "0.0.0")
+
+local_path_override(
+    module_name = "aspect_rules_js",
+    path = "../..",
+)
+
+node = use_extension("@rules_nodejs//nodejs:extensions.bzl", "node")
+
+node.toolchain(
+    name = "nodejs",
+)
+
+use_repo(node, "nodejs_toolchains")
+
+use_repo(node, "nodejs_linux_amd64")
+use_repo(node, "nodejs_linux_arm64")
+use_repo(node, "nodejs_darwin_amd64")
+use_repo(node, "nodejs_darwin_arm64")
+
+register_toolchains(
+    "@nodejs_toolchains//:all"
+)
+
+npm = use_extension("@aspect_rules_js//npm:extensions.bzl", "npm")
+
+npm.npm_translate_lock(
+    name = "npm",
+    bins = {
+        # derived from "bin" attribute in node_modules/typescript/package.json
+        "typescript": [
+            "tsc=./bin/tsc",
+            "tsserver=./bin/tsserver",
+        ],
```

Comment:
Note the alternative form we have to input this under a bzlmod extension.

Reply:
makes sense 👍